apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/StorageFixture.cs

```diff
@@ -288,6 +288,41 @@ private void CreateAndPopulateReadBucket()
 
         internal void UnregisterBucket(string bucket) => _bucketsToDelete.Remove(bucket);
 
+        internal async Task EventuallyAsync(Func<Task> action, TimeSpan delayUntilGuaranteedConsistency)
```

Comment:
Having looked at later commits:

- The calling code might be clearer if we put the timespan first
- We might want to add an overload taking an Action. (That could call this overload just by calling the action and then returning a completed task.)